Core/Packages/DeprecationInfo.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using NuGet.Versioning;
+
+namespace NuGetPe
+{
+    public class DeprecationInfo
+    {
+        public string? Message { get; set; }
+        public IEnumerable<string>? Reasons { get; set; }
```

Comment:
I don't think IEnumerable's should ever be nullable. At best, it can return an empty enumerable.

Reply:
Yes, I did it mostly because of the nullable warnings.

I can add
```c#
#pragma warning disable CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable.
````
around them.

Reply:
Yeah, I wish the code analysis would be better for DTO's. The pragma works until they improve it.